src/custom-surf/components/cim/ImpactedObjects.tsx

```diff
@@ -0,0 +1,294 @@
+import {
+    EuiBadge,
+    EuiBasicTable,
+    EuiButton,
+    EuiButtonIcon,
+    EuiFlexGrid,
+    EuiFlexItem,
+    EuiForm,
+    EuiLink,
+    EuiModal,
+    EuiModalBody,
+    EuiModalFooter,
+    EuiModalHeader,
+    EuiModalHeaderTitle,
+    EuiScreenReaderOnly,
+    EuiTitle,
+    RIGHT_ALIGNMENT,
+} from "@elastic/eui";
+import { HorizontalAlignment } from "@elastic/eui/src/services/alignment";
+import ImsCircuitInfo from "custom/components/cim/ImsCiruitInfo";
+import { ImpactedObject, ServiceTicketImpactedObjectImpact, ServiceTicketWithDetails } from "custom/types";
+import { isDate } from "lodash";
+import React, { Fragment, ReactNode, useContext, useState } from "react";
+import { WrappedComponentProps, injectIntl } from "react-intl";
+import ReactSelect from "react-select";
+import { getReactSelectTheme } from "stylesheets/emotion/utils";
+import ApplicationContext from "utils/ApplicationContext";
+import { Option } from "utils/types";
+import { isEmpty } from "utils/Utils";
+
+interface IProps extends WrappedComponentProps {
+    ticket: ServiceTicketWithDetails;
+    updateable: boolean;
+    mode: "withSubscriptions" | "withoutSubscriptions";
```

Comment:
Think it would be better to have a boolean for `withSubscriptions`

Reply:
Fixed